main.go

```diff
@@ -223,10 +227,13 @@ func main() {
 	ctx := context.Background()
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
-	ctx = context.WithValue(ctx, hostKey, os.Getenv("TODO_MONGO_HOST"))
-	ctx = context.WithValue(ctx, usernameKey, os.Getenv("TODO_MONGO_USERNAME"))
-	ctx = context.WithValue(ctx, passwordKey, os.Getenv("TODO_MONGO_PASSWORD"))
-	ctx = context.WithValue(ctx, databaseKey, os.Getenv("TODO_MONGO_DATABASE"))
+	ctx = context.WithValue(ctx, hostKey, os.Getenv("MONGODB_HOST"))
+	ctx = context.WithValue(ctx, usernameKey, os.Getenv("MONGODB_USER"))
+	ctx = context.WithValue(ctx, passwordKey, os.Getenv("MONGODB_PASS"))
+	ctx = context.WithValue(ctx, databaseKey, os.Getenv("MONGODB_DATABASE"))
+	ctx = context.WithValue(ctx, useSrvKey, os.Getenv("MONGODB_USESRV"))
+	fmt.Printf("%T", ctx)
+	fmt.Println("Context:", ctx)
```

Comment:
Were these for debugging or intentional? If they’re for debugging can you remove them?

Reply:
Yes - will clean those up... also - forgot that I renamed the env variables... not necessary... I will switch those back as well.